tidb-cloud/tidb-cloud-connect-aws-dms.md

```diff
@@ -0,0 +1,139 @@
+
+## Create TiDB Cloud DMS endpoints
+
+For connectivity, the steps for using TiDB Cloud clusters as a source or as a target are similar, but DMS does have some different database setting requirements for source and target. For more information, see [Using MySQL as a source](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.MySQL.html) or [Using MySQL as a target](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Target.MySQL.html). When using a TiDB Cloud cluster as a source, you can only **Migrate existing data** because TiDB does not support MySQL binlog.
+
+1. Go to the [Endpoints](https://console.aws.amazon.com/dms/v2/home#endpointList) page in the AWS DMS console, and switch to the corresponding region.
+
+    ![Create endpoint](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-connect-3.png)
+
+2. Click **Create endpoint** to create the target database endpoint.
+
+3. Select **Source endpoint** or **Target endpoint**.
+
+4. Fill in the **Endpoint identifier** and ARN, and then select **Source engine** or **Target engine** as **MySQL**.
+
+5. Choose **Provide access information manually** and fill in cluster information:
+
+    <SimpleTab>
+
+    <div label="TiDB Serverless">
+
+    - **Server name**: `HOST` of TiDB Serverless cluster.
+    - **Port**: `PORT` of TiDB Serverless cluster.
+    - **User name**: User of TiDB Serverless cluster for migration. Make sure it meets DMS requirements.
+    - **Password**: Password of the TiDB Serverless cluster user.
+    - **Secure Socket Layer (SSL) mode**: If you are connecting via public endpoint, it is highly recommended to set the mode to **verify-full** to ensure transport security. If you are connecting via private endpoint, you can set the mode to **none**.
+    - **CA certificate**: [ISRG Root X1 certificate](https://letsencrypt.org/certs/isrgrootx1.pem). For more information, see [TLS Connections to TiDB Serverless](/tidb-cloud/secure-connections-to-serverless-clusters.md).
+
+    </div>
+
+    <div label="TiDB Dedicated">
+
+    - **Server name**: `HOST` of TiDB Dedicated cluster.
+    - **Port**: `PORT` of TiDB Dedicated cluster.
+    - **User name**: User of TiDB Dedicated cluster for migration. Make sure it meets DMS requirements.
+    - **Password**: Password of TiDB Dedicated cluster user.
+    - **Secure Socket Layer (SSL) mode**: If you are connecting via public endpoint, we highly recommend setting it to **verify-full** to ensure transport security. If you are connecting via private endpoint, you can set it to **none**.
+    - **CA certificate**: Get the CA certificate according to [TLS Connections to TiDB Dedicated](/tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md).
+
+    </div>
+    </SimpleTab>
+
+     ![Provide access information manually](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-connect-4.png)
+
+6. If the endpoint to be created is a **Target endpoint**, set **Extra connection attributes** to `Initstmt=SET FOREIGN_KEY_CHECKS=0;`.
```

Comment:
```suggestion
6. If you create the endpoint as a **Target endpoint**, expand the **Endpoint settings** section, select the **Use endpoint connection attributes** checkbox, and then set **Extra connection attributes** to `Initstmt=SET FOREIGN_KEY_CHECKS=0;`.
```

It's hard to find the **Use endpoint connection attributes** field, because the **Endpoint settings** section is collapsed by default and we need to select **Use endpoint connection attributes** first.